File: FilRougeIb/FilRougeIb/Answer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FilRougeIb
{
    class Answer
    {
    //Attributs

        public long? IdAnswer { get; set; }
        public string AnswerText { get; set; }
        public bool Result { get; set; }
        public User Candidat { get; set; }
        public Question Question { get; set; }



        public Answer( string answerText, bool result, User candidat, Question question, long? idAnswer)
        {
            this.IdAnswer = idAnswer;
            this.AnswerText = answerText;
            this.Result = result;
            this.Candidat = candidat;
            this.Question = question;
        }

        //Constructors






    }








}

File: angularFilRougeIb/src/app/_models/answer.ts
import { time } from "console";
import { tmpdir } from "os";

export class Answer{

  private _idAnswer : number;
  private _answer : string;
  private _result : boolean;

  constructor(answer : string, result? : boolean, idAnswer? : number){
    this._idAnswer = idAnswer;
    this._answer = answer;
    this._result = result;
  }

  get idAnswer() : number {
    return this._idAnswer;
  }

  get answer() : string {
    return this._answer;
  }

  set answer(answer : string){
    this._answer = answer;
  }

  get result() : boolean {
    return this._result;
  }

  set result(result : boolean){
    this._result = result;
  }
}
